packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -146,47 +151,28 @@ Function Get-Java
     if ($javaPath -eq '') { Write-Error "Unable to determine the path to java.exe"; return $null }
     if ($javaCMD -eq '') { $javaCMD = "$javaPath\bin\java.exe" }
     if (-not (Test-Path -Path $javaCMD)) { Write-Error "Could not find java at $javaCMD"; return $null }
-  
+ 
     $ShellArgs = @()
 
+    Write-Verbose "Java detected at '$javaCMD'"
+    Write-Verbose "Java version detected as '$javaVersion'"
```

Comment:
I've not noticed before that we don't have any Java checking here (to see that it's a JRE and version that we support). Did the Windows scripts never have that?